components/script/animations.rs

```diff
@@ -76,6 +75,14 @@ impl Animations {
         self.pending_events.borrow_mut().clear();
     }
 
+    pub(crate) fn animations_present(&self) -> bool {
+        let has_running_animations = self.has_running_animations.get();
+        let has_pending_events = !self.pending_events.borrow().is_empty();
+        // is_animation_present should be true until all pending
+        // animation events are delivered.
```

Comment:
This comment used to explain that the message wasn't sent while there were pending events, but now it makes a bit less sense since when reading this method you don't know what it is used for. Probably it should be moved into a rustdoc comment for the method like:

```
/// Returns true if animations are running and there are no pending animations events to deliver.
/// This is used to update the animation state in the `Constellation`.
```